nielslange/woo-test-environment
# Don't stop the command execution when the theme has no sidebar

The goal of this PR is to allow the command execution to finish even when there's no sidebar.

Fixes https://github.com/nielslange/woo-test-environment/issues/2

### Testing
1. On a fresh installation, enable the `Twenty twenty two` theme (`wp theme activate twentytwentytwo`)
2. Run the command: ` wp woo-test-environment setup`
3. Make sure the command finishes successfully even if the log shows:
<img width="379" alt="Screenshot 2022-07-05 at 15 17 02" src="https://user-images.githubusercontent.com/186112/177336873-8f126d53-fe91-4da7-8f6f-4d16245bc3c0.png">
